tests/cds-rdm/test_full_migration.py

```diff
@@ -151,6 +155,113 @@ def multiple_versions_with_cs(record):
     }
 
 
+def multiple_custom_fields(record):
+    """2207587"""
+    dict_rec = record.to_dict()
+    assert "custom_fields" in dict_rec
+    assert dict_rec["custom_fields"] == {
+        "cern:accelerators": [
+            {"id": "CERN AD", "title": {"en": "CERN AD"}},
+            {"id": "CERN AD", "title": {"en": "CERN AD"}},
+        ],
+    }
+
+
+def file_missing(record):
+    """2783103"""
+    dict_rec = record.to_dict()
+    assert "files" in dict_rec
+    assert dict_rec["files"]["enabled"] == False
+    assert dict_rec["files"]["count"] == 0
+    assert "media_files" in dict_rec
+    assert dict_rec["media_files"]["enabled"] == False
+    assert dict_rec["media_files"]["count"] == 0
+
+
+def file_restricted(record):
+    """2783104"""
+    dict_rec = record.to_dict()
+    assert "access" in dict_rec
+    assert dict_rec["access"]["record"] == "public"
+    assert dict_rec["access"]["files"] == "restricted"
+    assert "files" in dict_rec
+    assert dict_rec["files"]["enabled"] == True
+    assert dict_rec["files"]["count"] == 1
+
+
+def irregular_exp_field(record):
+    """2046076"""
+    dict_rec = record.to_dict()
+    assert "custom_fields" in dict_rec
+    assert dict_rec["custom_fields"] == {
+        "cern:accelerators": [
+            {"id": "CERN AD", "title": {"en": "CERN AD"}},
+        ],
+    }
+
+
+def custom_affiliation(record):
+    """2041388"""
+    dict_rec = record.to_dict()
+    assert "affiliations" in dict_rec["metadata"]["creators"][0]
+    assert dict_rec["metadata"]["creators"][0]["affiliations"] == [
```

Comment:
could you check here also that there is no ror ID in the affiliation dict? otherwise it is not a custom affiliation